database/seeds/UsersTableSeeder.php

```diff
@@ -55,6 +56,30 @@ private function createCollegist()
         $user->roles()->attach(Role::getId(Role::PRINTER));
         $user->roles()->attach(Role::getId(Role::INTERNET_USER));
         $user->internetAccess->setWifiUsername();
+        PersonalInformation::create([
```

Comment:
Please create a factory here. If we write a test, we would use it anyway and it's safer to see randomly generated data. These are pretty simple, you can check the others in /factories/.